components/pd_client/src/client.rs

```diff
@@ -322,12 +348,34 @@ impl PdClient for RpcClient {
 
     fn watch_global_config(
         &self,
-    ) -> Result<grpcio::ClientSStreamReceiver<pdpb::WatchGlobalConfigResponse>> {
+        config_path: String,
+    ) -> PdFuture<tikv_util::mpsc::Receiver<Vec<GlobalConfigItem>>> {
         use kvproto::pdpb::WatchGlobalConfigRequest;
-        let req = WatchGlobalConfigRequest::default();
-        sync_request(&self.pd_client, LEADER_CHANGE_RETRY, |client, _| {
-            client.watch_global_config(&req)
-        })
+        let mut req = WatchGlobalConfigRequest::default();
```

Comment:
Need to set the watch version here, so we can ensure watch won't missing any changes after the last call of load